driver/src/test/java/org/example/driver/unit/sevice/CarServiceTest.java

```diff
@@ -0,0 +1,305 @@
+package org.example.driver.unit.sevice;
+
+import org.example.driver.constants.ExceptionConstants;
+import org.example.driver.dto.create.CarCreateEditDto;
+import org.example.driver.dto.read.CarReadDto;
+import org.example.driver.entity.Car;
+import org.example.driver.entity.Driver;
+import org.example.driver.entity.enumeration.Gender;
+import org.example.driver.exception.car.CarNotFoundException;
+import org.example.driver.exception.car.DuplicatedCarNumberException;
+import org.example.driver.mapper.CarMapper;
+import org.example.driver.repository.CarRepository;
+import org.example.driver.repository.DriverRepository;
+import org.example.driver.service.CarService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class CarServiceTest {
+
+    private static final Long DEFAULT_ID = 1L;
+
+    @Mock
+    private CarRepository carRepository;
+
+    @Mock
+    private CarMapper carMapper;
+
+    @Mock
+    private MessageSource messageSource;
+
+    @Mock
+    private DriverRepository driverRepository;
+
+    @InjectMocks
+    private CarService carService;
+
+    private Car defaultCar;
+    private CarCreateEditDto createCar;
+    private CarReadDto readCar;
+    private Driver driver;
+
+    @BeforeEach
+    void init() {
+        driver = new Driver(DEFAULT_ID, "name", "name@gmail.com", "+375331234567", Gender.MALE, defaultCar, 5.);
```

Comment:
Лучше использовать полную форму записи десятичной дроби.
```suggestion
        driver = new Driver(DEFAULT_ID, "name", "name@gmail.com", "+375331234567", Gender.MALE, defaultCar, 5.0);
```